src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -847,8 +848,14 @@ ServerHandlerImp::processRequest(
          */
         if (role != Role::IDENTIFIED && role != Role::PROXY)
         {
-            forwardedFor.clear();
-            user.clear();
+            // Simulate the clearing of string_view through a swap. Is this a
```

Comment:
I'm not comfortable with this method. Since these strings (`user` and `forwardedFor`) are being modified, they are not suitable candidates for `std::string_view`. `boost::string_view` has a `clear()` member function, until std. Should I retain the original version or change this occurence to `std:string` instead? @scottschurr

Reply:
`std::string_view` doesn't have a `clear()` method, but it does have a `remove_suffix()` method: https://en.cppreference.com/w/cpp/string/basic_string_view/remove_suffix

I think you might be able to use use...
```
forwardedFor.remove_suffix(firwardedFor.size())
```
What do you think?

Reply:
Yes thanks, this looks much cleaner 👍  I'll incorporate this change.